web/src/lib/elements/ui/dataTable/data-table.svelte

```diff
@@ -114,15 +130,47 @@
     .filter(([, hide]) => !hide)
     .map(([id]) => id);
 
-  const hidableCols = ['status', 'email', 'amount'];
+  const hidableCols = keys.slice(1);
+
+  async function getData() {
+    let sortKey: SortKey = $sortKeys[0];
+    let sortString;
+    if (sortKey) {
+      sortString = sortKey.id + ',' + sortKey.order;
```

Comment:
```suggestion
      sortString = `${sortKey.id},${sortKey.order}`;
```
Template string are awesome! You should use them :)